jekyll/_cci2/hello-world-macos.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: classic-docs
+title: "Hello World On MacOS"
+short-title: "Hello World On MacOS"
+description: "First macOS project on CircleCI 2.0"
+categories: [getting-started]
+order: 4
+---
+
```

Comment:
awesome! thanks for doing this. 
Let's also add this to the sidebar.yml in the getting started menu under Introduction
add add it to the demo-apps doc and tutorials doc

Reply:
Done!